tests/roundtrips.py

```diff
@@ -26,29 +26,35 @@
 ]
 
 opt_out = {
-    "asset3d": ["cpp", "py", "rust"],  # Don't need it, API example roundtrips cover it all
-    "bar_chart": ["cpp", "py", "rust"],  # Don't need it, API example roundtrips cover it all
-    "clear": ["cpp", "py", "rust"],  # Don't need it, API example roundtrips cover it all
-    "mesh3d": ["cpp", "py", "rust"],  # Don't need it, API example roundtrips cover it all
-    "scalar": ["cpp", "py", "rust"],  # TODO(jleibs)
-    "series_line": ["cpp", "py", "rust"],  # TODO(jleibs)
-    "series_point": ["cpp", "py", "rust"],  # TODO(jleibs)
+    "asset3d": ["cpp", "python", "rust"],  # Don't need it, API example roundtrips cover it all
+    "axes3d": ["cpp", "python", "rust"],  # TODO(#6525)
```

Comment:
This has an API snippet round trip trip as well so can match the above. I don't think we need separate round-trip test for this one.